common/supplemental/metaZones.xml

```diff
@@ -645,6 +645,9 @@ For terms of use, see http://www.unicode.org/copyright.html
 			<timezone type="America/Punta_Arenas">
 				<usesMetazone to="2016-12-03 23:00" mzone="Chile"/>
 			</timezone>
+			<timezone type="America/Coyhaique">
+				<usesMetazone to="2025-03-19 23:00" mzone="Chile"/>
```

Comment:
What is the new metazone?

Reply:
There is no new metazone. This puts the new timezone in the existing metazone so it has the same history as the timezone it was split off of.

Reply:
I modeled this after `America/Punta_Arenas`, which left "Chile Time" in 2016 to do its own thing (permanent DST). It doesn't have a metazone since then.

Reply:
This was probably fine when `America/Punta_Arenas` was the only time zone in the metazone, but now both time zones are in the same metazone, should they use the same display name.?

Reply:
There is no official (https://www.diariooficial.interior.gob.cl/publicaciones/2025/03/20/44104/01/2624263.pdf) or unofficial (https://www.latercera.com/nacional/noticia/tras-consulta-ciudadana-region-de-aysen-mantendra-el-horario-de-verano-durante-todo-el-ano/) name for this "timezone" (non-tech meaning) that would induce a new metazone.

Reply:
Should it be Chile Summer Time ?

Reply:
No

Reply:
The discussion was whether to wait for it to develop a name instead of inventing one unless there's a very strong case for why it needs one immediately

Reply:
`windowsZones.xml` uses "Magallanes Standard Time" for `America/Punta_Arenas` [1]. Though there seem to be no links to sources.

[1] https://github.com/unicode-org/cldr/commit/f2ef1270a1cd8554f2d80995435a2c700d5abbd3

Reply:
>Rule	Chile	2019	max	-	Apr	Sun>=2	3:00u	0	-
>Rule	Chile	2023	max	-	Sep	Sun>=2	4:00u	1:00	-

Chile rule uses daylight saving time from Sep to Apr.

>Zone America/Coyhaique	-4:48:16 -	LMT	1890
>...
>			-4:00	Chile	%z	2025 Mar 20
>			-3:00	-	%z

Until March 20 in this local uses UTC-4:00 + Daylight saving +1:00 = UTC-3:00. With the rule, the rule "-4:00 Chile" should cease 2025-03-20T00:00:00 in local time. And local time at that point is UTC-3:00. So the actual switch time in UTC should be 2025-03-20T03:00:00Z. I think the metazone mapping should end - 

<usesMetazone to="2025-03-20 03:00" mzone="Chile"/>

instead of 

<usesMetazone to="2025-03-19 23:00" mzone="Chile"/>

Reply:
I've followed what had been done for `America/Punta_Arenas`, which was also observing DST when its change happened: 

> Rule	Chile	2016	2018	-	Aug	Sun>=9	4:00u	1:00	-

> Zone America/Punta_Arenas -4:43:40 -	LMT	1890
...
			-4:00	Chile	%z	2016 Dec  4
			-3:00	-	%z

It is modeled as leaving the metazone at 2016-12-03 23:00.

It's all pretty arbitrary. The first instant where Coyhaique actually diverges from Santiago is 2025-04-05T03:00:00Z, I don't know how TZDB came up with these transition dates.

Reply:
Second this. It seems that 20th March is just the date when the law was published: https://github.com/eggert/tz/blob/771a14bf500f8ee8d8ab32e86d19dd3998a343d4/southamerica#L1353

Reply:
Found something interesting - https://unicode-org.atlassian.net/browse/CLDR-10089

I cannot remember what was done. The change was introduced by tzdata 2017a, and later, I pointed out the metazone mapping data should be corrected - and it looks I fixed the problem somewhere - probably in a release branch??

Reply:
Let's use the correct transition time aligned to TZDB for Coyhaique at least. Punta_Arenas seems also wrong, but it has been there for years.. If you don't want to touch it, we can update it later.

Reply:
ok

Reply:
@robertbastian Thanks.

```
<timezone type="America/Punta_Arenas">
	<usesMetazone to="2016-12-03 21:00" mzone="Chile"/>
</timezone>

```

I mentioned Punta_Arenas should swithc at 21:00, but it should be actually `2016-12-04 03:00` instead. Anyway, you did not make the change in this PR, and it's fine. I approved the changes.